docs/topics/jvm/jvm-create-project-with-spring-boot.md

```diff
@@ -0,0 +1,280 @@
+[//]: # (title: Create a Spring Boot project with Kotlin)
+[//]: # (description: Create a Spring Boot application with Kotlin using IntelliJ IDEA.)
+
+<microformat>
+    <p>This is the first part of the <strong>Getting started with Spring Boot and Kotlin</strong> tutorial:</p><br/>
+    <p><img src="icon-1.svg" width="20" alt="First step"/> <strong>Create a Spring Boot project with Kotlin</strong><br/><img src="icon-2-todo.svg" width="20" alt="Second step"/> Add a data class to the Spring Boot project<br/><img src="icon-3-todo.svg" width="20" alt="Third step"/> Add database support for Spring Boot project<br/><img src="icon-4-todo.svg" width="20" alt="Fourth step"/> Use Spring Data CrudRepository for database access<br/></p>
+</microformat>
+
+The first part of the tutorial shows you how to create a Spring Boot project in IntelliJ IDEA using Project Wizard.
+
+## Before you start
+
+Download and install the latest version of [IntelliJ IDEA](https://www.jetbrains.com/idea/download/index.html).
```

Comment:
```suggestion
Download and install the latest version of [IntelliJ IDEA Ultimate Edition](https://www.jetbrains.com/idea/download/index.html).
```

Reply:
Otherwise, I would download a Community Edition and then would be upset to see that I should have downloaded an Ultimate.